src/utils/generate_rml_pot.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import argparse
+import collections
+import datetime
+import re
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-s", action="store_true", help="sort strings")
+parser.add_argument("file", nargs="*")
+argv = parser.parse_args()
+
+translation_dict = collections.defaultdict(list)
+errors = 0
+
+def process_file(rml_file):
+    global translation_dict
+    rml_content = rml_file.read()
+    rml_file.close()
+    errors = 0
+    def error(msg):
+        print(f"Error on line {line}: {msg}", file=sys.stderr)
+        global errors
+        errors += 1
+    text_start = None
+    line = 1
+    line_count_position = 0
+    for m in re.finditer("<(/?)translate([^>]*)>", rml_content, re.DOTALL | re.IGNORECASE):
+        line += rml_content.count("\n", line_count_position, m.start())
+        line_count_position = m.start()
+        if m.group(1) == '/': # closing tag
+            if text_start is None:
+                error('unmatched </translate>')
+            else:
+                content = rml_content[text_start:m.start()]
+                if content != content.strip():
+                    error("leading or trailing whitespace in translated string")
+                if "\n" in content:
+                    error("translated string should not contain newlines")
+                for empty_element in re.finditer(r"<(\w+)(\s*/\s*)>", content):
+                    if empty_element.group(2) != " /":
+                        correct = f"<{empty_element.group(1)} />"
+                        error(f"empty element should be formatted like {correct} to match libRocket's GetInnerRML()")
```

Comment:
I think I got things working so that you no longer have to worry about matching the formatting.

Reply:
> Unfortunately I forgot to push the latest version when I was last working on this. There is another commit needed which should be squashed with the one: [8bed0e9](https://github.com/Unvanquished/Unvanquished/commit/8bed0e9e91a0a8d32e4e273ba577e1ce307a83ba). I could port it to RmlUi myself if you like.

If you can do that it would be kind. You can push to this branch, and I'll squash after that.